commit 5743891761f99bdfe6bf89956dab3a0e4c8a0fd0
Author: Steve Dickson <steved@redhat.com>
Date:   Wed Jun 27 12:42:05 2018 -0400

    Revert "gssd.c: Remomved a couple of warning errors"
    
    This reverts commit 98118f5e382cdeb1acf25370768d5dfdc254ba5d.
    Added back the 'm' character which cause memory to be
    allocated for the string convertions.
    
    Signed-off-by: Steve Dickson <steved@redhat.com>

diff --git a/utils/gssd/gssd.c b/utils/gssd/gssd.c
index 7b21ee2..2c14e5f 100644
--- a/utils/gssd/gssd.c
+++ b/utils/gssd/gssd.c
@@ -279,16 +279,16 @@ gssd_read_service_info(int dirfd, struct clnt_info *clp)
 	 * (commit bf19aacecbeebccb2c3d150a8bd9416b7dba81fe)
 	 */
 	numfields = fscanf(info,
-			   "RPC server: %s\n"
-			   "service: %s (%d) version %d\n"
-			   "address: %s\n"
-			   "protocol: %s\n"
-			   "port: %s\n",
-			   (char *)&server,
-			   (char *)&service, &program, &version,
-			   (char *)&address,
-			   (char *)&protoname,
-			   (char *)&port);
+			   "RPC server: %ms\n"
+			   "service: %ms (%d) version %d\n"
+			   "address: %ms\n"
+			   "protocol: %ms\n"
+			   "port: %ms\n",
+			   &server,
+			   &service, &program, &version,
+			   &address,
+			   &protoname,
+			   &port);
 
 
 	switch (numfields) {
